modules/ztp-configuring-spoke-at-remote-site.adoc

```diff
@@ -0,0 +1,162 @@
+----
+
+. Open a browser and log in to the edge cluster configuration user interface at the following URL with the supplied `kubeadmin` `username` and `password`:
++
+[source,terminal]
+----
+$ https://edge-cluster-setup.foobar-edge-cluster.domain.com
+----
++
+[NOTE]
+====
+This `kubeadmin` username and password was created at the factory and should have been supplied to you. Only one user is initially created.
+====
+
+. Click **Continue**.
+
+. Step through the screens to complete the initial setup.
+
+.. In the first two screens create a new user account by entering a `username` and `password` when prompted.
++
+[NOTE]
+====
+This new user account is granted `cluster-admin` privileges and should be used rather than the factory created `kubeadmin` account.
+====
+
+.. In the **API** screen assign the IP address that will be used for API traffic. A default value is assigned but you are free to update this.
+
+.. In the **Ingress** screen assign the IP address that will be used for new routes and traffic managed by the ingress controller. A default value is assigned but you are free to update this.
+
+.. Optional: Under Domain create unique URLs for the setup and console URLs for your edge cluster.
+
+.. Click **Download** in the **SSH** screen and download the edge cluster private SSH key.
++
+[NOTE]
+====
+You need this to access the nodes of the edge cluster.
+====
+
+.. Click **Finish setup**.
++
+When complete the screen switches to:
++
+.Edge cluster setup
+image::ztp-edge-cluster-setup1.png[Edge cluster setup]
+
+.. Selecting **OpenShift console** brings you direct to the web console.
+
+.. Under **Settings** you have the option to change the values of the **API address**, **Ingress address** and the previously configured **Domain**.
++
+[NOTE]
+====
+At this stage you will not be prompted for a username and password as you are already logged in as `kubeadmin`.
+====
+
+.. Click **Log out** in the top right hand corner.
+
+.Verification and final setup
+
+. Log in to the cluster again.
+
+. Select the newly created identity provider `ztpfw-htpassd-idp`.
+
+. In the cluster log in screen enter the `username` and `password` created in step 7a.
+
+. After you access the cluster, register your cluster subscription with the following steps:
+
+.. link:https://cloud.redhat.com/[Log in to the console] to register the disconnected OpenShift cluster. See link:https://access.redhat.com/solutions/4930131[How to register disconnected OpenShift Container Platform 4 cluster on cloud.redhat.com] for details.
+
+.. Obtain the pull secret from link:https://console.redhat.com/openshift/downloads[Pull secret], which can be found under `Tokens`.
+
+.. Change the global pull secret. Follow the guidance link:https://access.redhat.com/solutions/4902871[How to change the global pull secret in OCP 4] to do that.
+
+Your cluster is now registered to Red Hat OpenShift Cluster Manager and entitled to Red Hat subscription management.
```

Comment:
I think we have attributes for OCM in the common-attributes.adoc that it would be good to use here.